### PR TITLE
maintainers: add gou4shi1

### PR DESCRIPTION
**Pull request by gou4shi1**
###### Description of changes

https://github.com/NixOS/nixpkgs/pull/212587#issuecomment-1407579438

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by gou4shi1** [comment]
@wegank Could you help to reveiw?

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -16424,4 +16424,10 @@
     github = "srounce";
     githubId = 60792;
   };
+  gou4shi1 = {
```
Comment (fabaff) [inline maintainers/maintainer-list.nix:16427]: Please add the entry in alphabetical (as stated in the file header) order to the list. The entries at the end of the file usually sneaked in when time is an issue.

**Comment by Shawn8901** [comment]
Hi, glad to have you onboard!
I am trying to help to help on some structural topics:
Likely it would have been enough if you integrate this commit to the other PR, but as first commit in the chain. Now you have the situation that your first PR does depend on the other and no one can merge the other until this is done and the first pr was rebased (to not have un-evaluatable commits in the history). 

Beside that you can likely squash commits together (on both PRs), the commit history should not contain the walk to the final change, but the change itself (possibly splitted in multiple commits, if multiple things that can stand alone without the other changes are done in a PR). The contribution guidelines (linked in the PR template) is possibly something you want to cross check including the part of the nixpkgs manual about submitting changes (linked there aswell).

**Comment by gou4shi1** [comment]
OK, I will add this commit into another PR when it's needed.

**Pull request closed without merging**